src/LinearizedControlProblem.cpp

```diff
@@ -60,8 +60,11 @@ void LinearizedControlProblem::add(TaskWithRequirementsPtr tr)
 void LinearizedControlProblem::remove(TaskWithRequirements * tr)
 {
   ControlProblem::remove(tr);
```

Comment:
The question is whether we want to throw when one try to remove a task that is not there or just ignore silently. If we want to throw, then ControlProblem::remove should be changed instead.

Reply:
As discussed, we agree that `ControlProblem::remove` (and thus `LinearizedControlProblem::remove`) should silently ignore when asked to remove a task that was not added